src/commcare_cloud/ansible/status_check.yml

```diff
@@ -0,0 +1,84 @@
+- hosts: all
+  gather_facts: no
+  become: no
+  tasks:
+
+  - name: Ping if hosts are up
+    ping:
+    tags:
+     - ping
+
+- hosts:
+   - postgresql
+   - elsaticsearch
+   - riakcs
+   - couchdb2
+   - kafka
+   - redis
+  gather_facts: no
+  tasks:
+
+  - name: Check if /opt/data is mounted
+    shell: grep '{{ encrypted_root }} ext4 rw' /etc/mtab
+    check_mode: no
+    tags:
+    - opt_data
+
+  - name: Check for already mounted encrypted root drive
+    shell: grep '{{ encrypted_root }} ecryptfs' /etc/mtab
+    failed_when: false
+    changed_when: false
+    check_mode: no
+    tags:
+    - ecryptfs
+
+  - name: Check /opt/data is not read-only
+    become: yes
+    file:
+      path: "{{ encrypted_root }}/ansible-test"
+      state: touch
+    register: opt_data_readable
+    tags:
+    - opt_data
+
+  - name: Remove the test file
+    become: yes
+    file:
+      path: /opt/data/ansible-test
+      state: absent
+    when: opt_data_readable
```

Comment:
I think this should be `opt_data_readable is success`. Same for `/home/cchq` below